Index: config.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\n    \"domainPattern\": \"example*.com\"\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.json b/config.json
--- a/config.json	(revision 0b4a6bcb99d0fb027f1156b4674d2ba37c8ff11a)
+++ b/config.json	(date 1750525093976)
@@ -1,3 +1,3 @@
 {
-    "domainPattern": "example*.com"
+    "domainPattern": "dizipal*.com"
 }
